README.md

```diff
@@ -1,2 +1,215 @@
-# BookFlix
-Blockbuster Booking is a platform where users can book tickets to movie premieres, cancel a booking, see the details of a premiere, and rate the premiere.
+<a name="readme-top"></a>
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
@Meltrust 
Thank you so much for the review, the folder has been deleted at this [commit](https://github.com/Lucash2022/BookFlix-Backend/commit/90d3a741b12af76ac317cbd74753ba3775670208)